src/components/BugsHistory/BugsHistory.scss

```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+
+.bugs-current {
```

Comment:
Do not add widget-specific classes as root-level. Group under `.panel--bugshistory`, e.g.:
```
.panel--bugshistory {
  .bugs-current {
  ...
  }
}
```